OWNERS

```diff
@@ -1,17 +1,19 @@
 reviewers:
-- skeeey
-- zhiweiyin318
+- Jaanki
 - qiujian16
+- skeeey
 - skitt
+- tpantelis
+- zhiweiyin318
```

Comment:
Why were all these lines shuffled around?

Reply:
Rearranged alphabetically

Reply:
Are they alphabetical? The rending GitHub is showing me is [j,q,s,t,z,a,d,s,v].

Reply:
This is split like this - all approvers first alphabetically and then all reviewers. The original setup was all approvers (non alphabetically) and then all reviewers (alphabetically)

Reply:
> The original setup was all approvers (non alphabetically) and then all reviewers (alphabetically)

That was only the result of adding people to the reviewers list and not the approvers list, there’s no underlying intent.